apps/mobile-wallet/src/navigation/RootStackNavigation.tsx

```diff
@@ -252,10 +250,15 @@ const AppUnlockModal = ({ initialRouteName }: Required<RootStackNavigationProps>
   useAutoLock(unlockApp)
 
   return (
-    <Modal visible={!!lastUsedWalletId && biometricsRequiredForAppAccess && !isWalletUnlocked} animationType="none">
-      <View style={{ backgroundColor: 'black', flex: 1 }}>
-        <AnimatedCirclesBackground isAnimated />
-      </View>
+    <Modal visible={!!lastUsedWalletId && biometricsRequiredForAppAccess && !isWalletUnlocked} animationType="fade">
```

Comment:
Hm, I remember that changing it to `fade` creates a weird artefact but I don't remember exactly how. I need to do a functional test.

Reply:
I don't see any artefact in the next PR 🤞